app.js

```diff
@@ -901,7 +901,7 @@ io.sockets.on('connection',function(socket){
                 openhab.status = 'online';
                 openhab.last_online = new Date;
                 openhab.openhabVersion = socket.handshake.openhabVersion;
-                openhab.myohVersion = socket.handshake.myohVersion;
+                openhab.clientVersion = socket.handshake.clientVersion;
```

Comment:
@digitaldan should we "add" it to the mongodb schema or remove the old entry in schema.
I think we can add it or what strategy do we wanna go with this?